backend/franky/service/stackoverflow/stackoverflow.py

```diff
@@ -42,12 +43,20 @@ def _get_user_id(self, user_name: str) -> str:
             raise StackoverflowServiceException('User not found')
         return str(user_id)
 
-    def _get_user_tags(self, user_id: str) -> List:
+    def _get_user_tags(self, user_id: str) -> Dict:
         payload = {'order': 'desc', 'sort': 'popular'}
         request_url = f'{self._user_url}/{user_id}/{self._tags_url}'
         response_data = self._call(request_url, payload)
-        tags_name = [tag['name'] for tag in response_data['items']]
-        return tags_name
+        tags = self._calculate_tags_weight(response_data['items'])
+        return tags
+
+    def _calculate_tags_weight(self, response_data: List) -> Dict:
+        try:
+            sum_tags = reduce(lambda a, b: a + b, [tag['count'] for tag in response_data])
+            tags = {tag['name']: tag['count'] / sum_tags for tag in response_data}
+        except TypeError:
```

Comment:
What is the case when `TypeError` is thrown? 

Btw if you catch the error please log it using `logging` package rather then ignoring it.